src/Queue.py

```diff
@@ -161,13 +163,24 @@ def randomPop() -> Tuple[List[BallChaser], List[BallChaser]]:
     return blueTeam, orangeTeam
 
 
+def getTop2() -> List[Document]:
+    top2 = []
+    sorted_MMRList = sorted(currQueue.all(), key=lambda x: (x[BallChaserKey.MMR]), reverse=True)
+    top2.extend([sorted_MMRList[0], sorted_MMRList[1]])
```

Comment:
Instead of using `extend` on an empty array, can we just initialize top2 to the first two elements of the sorted list?
Like:
```py
sorted_MMRList = sorted(currQueue.all(), key=lambda x: (x[BallChaserKey.MMR]), reverse=True)
top2 = sorted_MMRList[0:2]
```

Reply:
If you do that ^ I wouldn't even make this a separate function to be honest. I would just put this code inside of captains pop. It doesn't look like you're re-using this anywhere. What do you think?